src/heaps/mutable_binary_heap.jl

```diff
@@ -212,6 +212,16 @@ top(h::MutableBinaryHeap) = h.nodes[1].value
 
 pop!{T}(h::MutableBinaryHeap{T}) = _binary_heap_pop!(h.comparer, h.nodes, h.node_map)
 
+"""
+    update!{T}(h::MutableBinaryHeap{T}, i::Int, v::T)
+
+### Args:
+* `h`: Heap derived from `AbstractMutableHeap`.
+* `i`: The handle to the element which needs to be updated.
+* `v`: The updated value.
+
+Updates the value of an element in heap `h` (referred to by the handle `i`) with the value `v`.
```

Comment:
Shouldn't "handle" be simply "index"? If so, I'd simply say "Replace the element at index `i` in heap `h` with `v`.